summerofcode/2023.md

```diff
@@ -285,3 +285,25 @@ If you are a project maintainer and consider mentoring during the GSoC 2023 cycl
 - Expected project size: 350 hours
 - Difficulty: Hard
 - Upstream Issue (URL): https://github.com/opencurve/curve/issues/2229
+
+### CRI-O
+
+#### Add additional log drivers to conmon-rs
+
+- Description: conmon-rs is a container monitor written in Rust, used by CRI-O to monitor a container's lifecycle. Part of its responsibilities is log forwarding--taking the output of the container and writing that output to various places. Currently, conmon-rs supports one format: the one required by the Kubernetes CRI. The goal of this proposal is to add new log formats from the list of standardized ones, like JSON, Splunk, Journald. 
+- Expected outcome: A JSON log driver and Journald log driver are added to conmon-rs. A stretch goal of adding a Splunk log driver is also within scope.
+- Recommended Skills: Rust, familiarity with containers
+- Mentor(s): Peter Hunt, haircommander, pehunt@redhat.com; Sascha Grunert, saschagrunert, sgrunert@redhat.com
+- Expected project size: 175 Hours
+- Difficulty: Medium
+- Upstream Issue (URL): https://github.com/containers/conmon-rs/issues/1126
+
+#### Add podman support in conmon-rs
+
+- Description: conmon-rs is a container monitor written in Rust, used by CRI-O to monitor a container's lifecycle. In order to declare stability of conmon-rs for CRI-O, many features have to be added for its sibling project: [podman](podman.io). The goal of this proposal is to add additional features that are required by podman to conmon-rs, so conmon-rs can be stabalied
```

Comment:
Can we add a few examples of the features that are required by podman? It would give a better idea to the students when they decide which project to choose.

Reply:
I think that we can probably merge as is and leave the addition of examples as a follow up PR.